docker/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM debian:9.6-slim
 
-# Default google-fluentd at the unstable version.
-# Use `docker build --build-arg REPO_SUFFIX=<CUSTOM_REPO_SUFFIX>` to build
-# images with a specific version of the package.
-ARG REPO_SUFFIX=unstable
+# Default google-fluentd at a specific version. This version should match the
+# lastest stable.
```

Comment:
s/lastest/latest/, though you could also just say:
"Default google-fluentd to the latest stable."

Reply:
Done